backend/.env.example

```diff
@@ -0,0 +1,4 @@
+PORT=8000
```

Comment:
ok if we change this might need to adjust the auth url's for dev also, but is currently going through client directly

Reply:
It's working now, can you please try the links I put on the PR description?